test/ode/ode_ssprk_tests.jl

```diff
@@ -325,6 +326,8 @@ for prob in test_problems_nonlinear
 	sim = test_convergence(dts, prob, alg)
 	@test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
 end
+#reverting back to original dts
+dts = 1 .//2 .^(8:-1:4)
 
 # for NDBLSRK124 to be in asymptotic range
 dts = 1 .//2 .^(7:-1:3)
```

Comment:
it changes right after anyways

Reply:
I made that as a precaution, In case somebody inserts some method there.

Reply:
dts are pretty specific to each case. I don't think there's a need to attempt this. It can safely be deleted.